text_extract.py

```diff
@@ -0,0 +1,29 @@
+import os
+import re
+from glob import glob
+import hwp5
+# from util import makedir
+
+class HWPExtractor():
+	def __init__(self):
+		# try:
+		# 	import hwp5
+		# except:
+		# 	print('Package "pyhwp" is requred to extract HWP files.')
+		# 	print('pip install --pre pyhwp')

```

Comment:
추후 개발할 내용은 "TODO" 주석을 활용

Reply:
Documentation 및 TODO 리스트 추가
```suggestion
		'''
		TODO:
		- add documentation
		- warning to instruct what version of pyhwp must be installed (not the released version)
		# try:
		# 	import hwp5
		# except:
		# 	print('Package "pyhwp" is requred to extract HWP files.')
		# 	print('pip install --pre pyhwp')
		'''
```